packages/core/src/Stores/client-store.js

```diff
@@ -491,6 +492,10 @@ export default class ClientStore extends BaseStore {
         return this.active_accounts.some(acc => acc.is_virtual === 0);
     }
 
+    get has_svg_account() {
+        return this.active_accounts.some(acc => acc.landing_company_shortcode === 'svg');
+    }
```

Comment:
@thisyahlen-deriv Please create a hook in the `@deriv/hooks` package for new logic instead of adding more logic into stores 🙏🏻

Reply:
Can do 🙏